### PR TITLE
#237 customer input

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #227 

image

Not sure if the styles are quite right here but thought to stop here. Will do something very similar for the ref input, so figured heres a good place to stop and get feedback before moving forward.

### CODE REVIEW

File: packages/common/src/ui/components/inputs/TextInput/InputWithLabelRow.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+interface InputWithLabelRowProps {
+  Input: ReactNode;
+}
+
+export const InputWithLabelRow: FC<InputWithLabelRowProps> = ({
+  Input = <BasicTextInput />,
+}) => {
+  return (
+    <Grid display="flex" alignItems="center" gap={1}>
+      <FormLabel sx={{ fontWeight: 'bold' }}>Customer name:</FormLabel>
```
Comment (mark-prins) [inline packages/common/src/ui/components/inputs/TextInput/InputWithLabelRow.tsx:14]: was going to point out that it needs a translation, but then noticed this is a shared component.. so will need the label to be passed in

File: packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx [hunk 2]
```diff
@@ -210,12 +212,34 @@ export const OutboundShipmentDetailViewComponent: FC = () => {
...
+        </Box>
+
+        <Box display="flex" flex={1} justifyContent="center">
+          <TabList value={currentTab} onChange={onChangeTab}>
+            <Tab value="general" label={t('label.general')} />
+            <Tab value="transport" label={t('label.transport')} />
+          </TabList>
+        </Box>
+
+        <Box display="flex" flex={1} justifyContent="flex-end" />
```
Comment (mark-prins) [inline packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx:242]: Is it time to put a save and cancel button on the bottom of this component 🤔

File: packages/mock-server/src/worker/handlers.ts [hunk 3]
```diff
@@ -108,7 +121,7 @@ export const serverError = graphql.query('error500', (_, response, context) =>
  * handler.
  */
 const batchMutationHandler = rest.post(
-  Environment.API_URL,
+  'http://localhost:4000',
```
Comment (mark-prins) [inline packages/mock-server/src/worker/handlers.ts:124]: are you having issues with the config too?

File: packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx (hunk 2 above)
Comment (joshxg) [reply packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx:243]: Ha yes I think so!

**Comment by joshxg** [comment]
Seems what I've added in the MSW to handle batched mutations means the items query wasn't falling through and hitting the server. So i've added an item handler in the worker for now and will look at doing a better solution for the mock worker.

Also fixed up the nits in the other PR in this one

**Pull request merged**